doc/Testing.md

```diff
@@ -98,6 +126,60 @@ For more documentation on testing, read the references listed at the bottom.
     }
     ```
 
+## Functionnal/Integration Testing
+
+- We use [integration_test](https://docs.flutter.dev/testing/integration-tests) to create functionnal/integration tests. You can create a test like this :
```

Comment:
Write integration_test**s** if that's the name of the package

Reply:
The name of the package is integration_test

Reply:
![image](https://github.com/nventive/FlutterApplicationTemplate/assets/72167772/844a1120-2764-4f58-9aaa-d80f0daee788)

Reply:
Oh ok, I got confused because of the url.